src/app/embed/adapters/feed.adapter.ts

```diff
@@ -83,7 +83,8 @@ export class FeedAdapter implements DataAdapter {
 
     if (typeof episodeGuid !== 'undefined') {
       for (let i = 0; i < items.length; i++) {
-        let itemGuid = this.getTagText(items[i], 'guid');
+        // If the item doesn't include a `guid`, fall back to RadioPublic's `rp:item-id` tag.
+        let itemGuid = this.getTagText(items[i], 'guid') || this.getTagText(items[i], 'rp:item-id');
```

Comment:
I think this probably belongs in the `draper.adapter.ts` subclass.  Something like...

- Add a `getItemGuid(item)` function to this class
- Change this line to `this.getItemGuid(items[i])`
- Override the function in the DraperAdapter, to have the `rp:item-id` fallback.